scripts/generate-react.js

```diff
@@ -17,12 +17,15 @@ export default ${component.name};
 
 declare const ${component.name}: MdiReactIconComponentType;
 export default ${component.name};
-`, () => `import { ComponentType } from 'react';
+`, () => `import { ComponentType, HTMLProps } from 'react';
 
-export interface MdiReactIconProps {
+type SVGProps = HTMLProps<SVGElement>
```

Comment:
Could we use `SVGProps<SVGSVGElement>` here instead of `HTMLProps<SVGElement>`.

Reply:
Done. thanks.